docs/indicators/indicators.md

```diff
@@ -52,4 +52,14 @@ List of indicator aggregate data currently available.
 | `barrier_reason_dontneed_other`           | Respondents who do not believe they need a COVID-19 vaccine because of other reasons which are not not listed.    | V6_7,V1  |
 | `informed_access`                         | Respondents who reported being very informed or moderately informed about how to be able to get a COVID-19 vaccine.	   | V13  |
 | `appointment_have`                        | Respondents who reported having an appointment to receive a COVID-19 vaccine.   | V15  |
-| `appointment_tried`                       | Respondents who reported having tried to get an appointment to receive a COVID-19 vaccine.   | V16  |
\ No newline at end of file
+| `appointment_tried`                       | Respondents who reported having tried to get an appointment to receive a COVID-19 vaccine.   | V16  |
+| `work_outside_home_1d`                       | Respondents who worked or went to school outside their home in the past 24 hours.	   | C0_1  |
+| `shop_1d`                       | Respondents who went to a market, grocery store, or pharmacy in the past 24 hours.	   | C0_2  |
+| `restaurant_1d`                       | Respondents who went to a bar, restaurant, or cafe in the past 24 hours.	   | C0_3  |
+| `spent_time_1d`                       | Respondents who spent time with someone who isn’t currently staying with them in the past 24 hours.	   | C0_4  |
+| `large_event_1d`                       | Respondents who attended an event with more than 10 people in the past 24 hours.	   | C0_5  |
+| `public_transit_1d`                       | Respondents who used public transit in the past 24 hours.	   | C0_6  |
+| `food_security`                       | Respondents who worried about will they have enough to eat in the next week.	   | D4  |
```

Comment:
can you change this one to `Respondents who reported being worried about having enough to eat in the next week`?